tests/test_request_scoped_k8s.py

```
@@ -0,0 +1,199 @@
+    # 0600 permissions: not readable/writable by group or others.
+    mode = os.stat(path).st_mode & 0o777
+    assert mode == 0o600
+
+    content = open(path).read()
+    assert "token: 'SECRET'" in content
+    assert "Bearer" not in content  # bearer prefix must be stripped
+    assert "server: 'https://api.example:6443'" in content
+    assert f"certificate-authority: '{ca}'" in content
+
+    # Two concurrent requests must get distinct kubeconfig files (no shared state).
+    env2, path2 = build_request_scoped_env(ctx)
```

Ruff (0.16.1)

[warning] Unpacked variable `env2` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)